install.sh

```diff
@@ -199,6 +199,28 @@ ops_install_qemu() {
   fi
 }
 
+ops_install_haxm() {
+    if [ "$OS" != "darwin" ]; then
+        return
+    fi
+
+    # check to see if a version of the package was previously
+    # installed or if the haxm kext was otherwise loaded. 
+    existing_install=`pkgutil --pkgs | grep haxm`
+    loaded_module=`kextstat | grep intelhaxm`
+    if [ ! -z $existing_install ] || [ ! -z $loaded_module ]; then
```

Comment:
The .mpkg in the .dmg does this check and also runs a script that checks to see if a binary is running (amongst other things). We do a quick check here to avoid having to go through the trouble of mounting the .dmg and such if we know the installer is just going to fail.